drivers/sdcard_spi/sdcard_spi.c

```diff
@@ -712,8 +696,8 @@ static sd_rw_response_t _write_data_packet(sdcard_spi_t *card, char token, const
 
     if (_transfer_bytes(card, data, 0, size) == size) {
 
-        uint16_t data__crc_16 = _crc_16(data, size);
-        char crc[sizeof(uint16_t)] = { data__crc_16 >> 8, data__crc_16 & 0xFF };
+        uint16_t data_crc16 = ucrc16_calc_be((const uint8_t *)data, size, UCRC16_CCITT_POLY_BE, 0);
```

Comment:
same as above. It does no harm, but for readability I'd remove the 'const' here.